setup.cfg

```diff
@@ -41,6 +41,7 @@ where = src
 [options.extras_require]
 dev =
     devtools==0.7.0
+    mock
```

Comment:
Let's specify version for now just to be compatible. I know that we wanted to remove them but that should be a part of another task

Reply:
Done